{"stats": {"errors": 0, "failures": 1, "passes": 6, "skipped": 0, "encoding": "UTF-8", "total": 7}, "results": [{"classname": "index_test:TestDistance", "name": "test_avenue_distance", "time": 0.0006566047668457031, "type": "success"}, {"classname": "index_test:TestDistance", "name": "test_distance", "time": 0.00047850608825683594, "type": "success"}, {"classname": "index_test:TestDistance", "name": "test_distance_all", "time": 0.0016703605651855469, "type": "success"}, {"classname": "index_test:TestDistance", "name": "test_distance_between_neighbors_squared", "time": 0.0011746883392333984, "type": "success"}, {"classname": "index_test:TestDistance", "name": "test_distance_with_neighbor", "time": 0.00043463706970214844, "type": "success"}, {"classname": "index_test:TestDistance", "name": "test_nearest_neighbors", "time": 0.010347127914428711, "type": "failure", "errtype": "builtins.AssertionError", "message": "Lists differ: [{'name': 'Bob', 'avenue': 5, 'street': 8, '[319 chars]504}] != [{'avenue': 5, 'distance': 1.0, 'name': 'Bob[87 chars]: 6}]\n\nFirst list contains 3 additional elements.\nFirst extra element 2:\n{'name': 'Natalie', 'avenue': 5, 'street': 4, 'distance': 4.123105625617661}\n\n  [{'avenue': 5, 'distance': 1.0, 'name': 'Bob', 'street': 8},\n-  {'avenue': 3, 'distance': 2.23606797749979, 'name': 'Steven', 'street': 6},\n?                                                                            ^\n\n+  {'avenue': 3, 'distance': 2.23606797749979, 'name': 'Steven', 'street': 6}]\n?                                                                            ^\n\n-  {'avenue': 5, 'distance': 4.123105625617661, 'name': 'Natalie', 'street': 4},\n-  {'avenue': 1, 'distance': 4.242640687119285, 'name': 'Suzie', 'street': 11},\n-  {'avenue': 6, 'distance': 5.385164807134504, 'name': 'Edgar', 'street': 13}]", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/site-packages/unittest2/case.py\", line 67, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/site-packages/unittest2/case.py\", line 625, in run\n    testMethod()\n  File \"/home/erickalexander/calculating-distance-lab-data-science-pilot/test/index_test.py\", line 32, in test_nearest_neighbors\n    {'avenue': 3, 'distance': 2.23606797749979, 'name': 'Steven', 'street': 6}])\n  File \"/opt/conda/lib/python3.6/site-packages/unittest2/case.py\", line 836, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/opt/conda/lib/python3.6/site-packages/unittest2/case.py\", line 1032, in assertListEqual\n    self.assertSequenceEqual(list1, list2, msg, seq_type=list)\n  File \"/opt/conda/lib/python3.6/site-packages/unittest2/case.py\", line 1014, in assertSequenceEqual\n    self.fail(msg)\n  File \"/opt/conda/lib/python3.6/site-packages/unittest2/case.py\", line 690, in fail\n    raise self.failureException(msg)\nAssertionError: Lists differ: [{'name': 'Bob', 'avenue': 5, 'street': 8, '[319 chars]504}] != [{'avenue': 5, 'distance': 1.0, 'name': 'Bob[87 chars]: 6}]\n\nFirst list contains 3 additional elements.\nFirst extra element 2:\n{'name': 'Natalie', 'avenue': 5, 'street': 4, 'distance': 4.123105625617661}\n\n  [{'avenue': 5, 'distance': 1.0, 'name': 'Bob', 'street': 8},\n-  {'avenue': 3, 'distance': 2.23606797749979, 'name': 'Steven', 'street': 6},\n?                                                                            ^\n\n+  {'avenue': 3, 'distance': 2.23606797749979, 'name': 'Steven', 'street': 6}]\n?                                                                            ^\n\n-  {'avenue': 5, 'distance': 4.123105625617661, 'name': 'Natalie', 'street': 4},\n-  {'avenue': 1, 'distance': 4.242640687119285, 'name': 'Suzie', 'street': 11},\n-  {'avenue': 6, 'distance': 5.385164807134504, 'name': 'Edgar', 'street': 13}]\n"}, {"classname": "index_test:TestDistance", "name": "test_street_distance", "time": 0.0002357959747314453, "type": "success"}]}